mage/import_test.go

```diff
@@ -187,3 +187,31 @@ func TestMageImportsOneLine(t *testing.T) {
 		t.Fatalf("expected: %q got: %q", expected, actual)
 	}
 }
+
+func TestMageImportsTaggedPackage(t *testing.T) {
+	stdout := &bytes.Buffer{}
+	stderr := &bytes.Buffer{}
+	inv := Invocation{
+		Dir:    "./testdata/mageimport/tagged",
+		Stdout: stdout,
+		Stderr: stderr,
+		List:   true,
+	}
+
+	code := Invoke(inv)
+	if code != 1 {
+		t.Fatalf("expected to exit with code 1, but got %v, stdout:\n%s\nstderr:\n%s", code, stdout, stderr)
+	}
+
+	actual := stderr.String()
+	expected := `
+Error parsing magefiles: error running "go list -f {{.Dir}}||{{.Name}} github.com/magefile/mage/mage/testdata/mageimport/tagged/pkg": exit status 1
+can't load package: package github.com/magefile/mage/mage/testdata/mageimport/tagged/pkg: build constraints exclude all Go files in /home/arve/Projects/mage/mage/testdata/mageimport/tagged/pkg
```

Comment:
Oh yeah, this test won't work as is... you could just look for a prefix of the string up to "exclude all Go files in" ... I don't think we need to worry that the path will be wrong.

Reply:
@natefinch Ooops, I see the problem now :man_facepalming: Thanks, I will fix it.